android_tests/lib/android/specs/common/command.rb

```diff
@@ -9,45 +9,45 @@ def before_first
   end
 
   t 'check all command no arg' do
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:shake).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:launch_app).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:close_app).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:reset).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:device_locked?).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:open_notifications).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:toggle_airplane_mode).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:current_activity).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:current_package).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_network_connection).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_performance_data_types).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_performance_data).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_system_bars).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_display_density).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:is_keyboard_shown).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:shake).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:launch_app).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:close_app).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:reset).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:device_locked?).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:open_notifications).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:toggle_airplane_mode).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:current_activity).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:current_package).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_network_connection).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_performance_data_types).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_performance_data).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_system_bars).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_display_density).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:is_keyboard_shown).must_equal true
```

Comment:
Yeah, this section can implement such as your suggestion.
BTW, I'd like to reduce control flow in test code. Anyway, I'll consider here and similar sections after passing tests.

Thanks!